xc/xc7/tests/mmcm/CMakeLists.txt

```diff
@@ -60,6 +60,10 @@ add_file_target(FILE mmcm_int_basys3_bottom.v SCANNER_TYPE verilog)
 add_file_target(FILE mmcm_buf_basys3_bottom.v SCANNER_TYPE verilog)
 add_file_target(FILE mmcm_ext_basys3_bottom.v SCANNER_TYPE verilog)
 add_file_target(FILE mmcm_int_frac_basys3_bottom.v SCANNER_TYPE verilog)
+add_file_target(FILE mmcm_int_frac_basys3_bottom.v SCANNER_TYPE verilog)
```

Comment:
This file is already added as a target. A double target addition will cause CMake to fail